nextflow_schema.json

```diff
@@ -235,7 +251,12 @@
                     "type": "string",
                     "default": "cut",
                     "description": "Masking mode for the pRESTO MaskPrimer step. Available: cut, mask, trim, tag.",
-                    "enum": ["cut", "mask", "trim", "tag"],
+                    "enum": [
+                        "cut",
+                        "mask",
+                        "trim",
+                        "tag"
```

Comment:
```suggestion
                        "cut",
                        "mask",
                        "tag",
                        "trim"
```

Reply:
This comes from the pipeline template, so best would be to fix it there, as otherwise I'm gonna have conflicts here next time 😄